src/common/dsc.h

```diff
@@ -296,6 +301,14 @@ typedef struct dsc
 		dsc_address = (UCHAR*) address;
 	}
 
+	void makeDecimalFixed(Firebird::DecimalFixed* address = NULL)
```

Comment:
@AlexPeshkoff shouldn't you set the scale in this function?

Reply:
Hops - this function is not used at all. Removed it.